schema/m3-enemies.schema.json

```diff
@@ -175,6 +175,20 @@
               false
             ]
           },
+          "grapplable": {
```

Comment:
Good idea. May I suggest an enum of possible values here as a quality of life improvement?